packages/signing-server/src/server.ts

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env node
+import yargs = require("yargs");
+import { Civil, TwoStepEthTransaction } from "@joincivil/core";
+import { infuraProvider } from "@joincivil/dev-utils";
+import { isValidAddress, toBuffer } from "ethereumjs-util";
+import * as Koa from "koa";
+import * as logger from "koa-logger";
+import * as route from "koa-route";
+import { ActionSigner } from "./lib/actionsigner";
+import BN = require("bn.js");
+
+const args = yargs
+  .env("SIGNING_SERVER")
+  .option("port", {
+    alias: "p",
+    describe: "server port to listen to",
+    requiresArg: true,
+    default: "8080",
+  })
+  .option("mnemonic", {
+    alias: "m",
+    describe: "mnemonic to generate private key",
+    requiresArg: true,
+    demandOption: true,
+  })
+  .option("nodeEndpoint", {
+    describe: "Ethereum node url",
+    requiresArg: true,
+    demandOption: true,
+  })
+  .strict().argv;
+
+const port = Number.parseInt(args.port, 10);
```

Comment:
this should probably be an environment variable, I think NODE_PORT is the most common

Reply:
Yargs enables env parsing.
`SIGNING_SERVER_PORT=1234` will work, similar for all other params